returnn/torch/engine.py

```diff
@@ -695,12 +709,18 @@ def eval_model(self, *, skip_already_evaluated: bool = False):
                         start_elapsed=step_end_time - eval_start_time,
                         log_memory_usage_device=self._device if self._log_memory_usage else None,
                     )
+
                     step_idx += 1
 
             assert step_idx > 0, f"No data in dataset {dataset_name!r}."
             accumulated_losses_dict = accumulated_losses_dict / accumulated_inv_norm_factors_dict
             accumulated_losses_dict = self._maybe_extend_losses_info(accumulated_losses_dict)
 
+            if self._tensorboard_writer:
+                # write losses/errors to tensorboard
+                for key, val in accumulated_losses_dict.items():
+                    self._tensorboard_writer.add_scalar(f"{dataset_name}/{key}", val, global_step=self.epoch)
```

Comment:
I wonder whether we should use `self.global_train_step` here instead.
```suggestion
                    self._tensorboard_writer.add_scalar(f"{dataset_name}/{key}", val, global_step=self.global_train_step)
```

We did it like that for TF. Also to have it comparable to the train scores. But I was also not so sure about that.

Other opinions? @NeoLegends? @Atticus1806? @JackTemaki? @vieting? @michelwi?

Reply:
There was overall not really much feedback on this question. So I would go with `global_train_step` here, to have it consistent to the training.